sdk/storage/Azure.Storage.Blobs/src/PartitionedDownloader.cs

```diff
@@ -354,6 +355,17 @@ await CopyToInternal(
             }
             finally
             {
+#pragma warning disable AZC0110
+                if (runningTasks?.Count > 0)
```

Comment:
This is a minor comment/suggestion. To my understanding, we don't really need to check for the size of the queue (tho we still need to check for null) since Task.WhenAll() can handle an empty queue and just not process. I think it might be a little cleaner without the size check but may introduce a little overhead by still calling Task.WhenAll() when the queue is empty? Feel free to ignore if the coding style standard doesn't align with this